woocommerce/woocommerce-android
# Add tag to zendesk with IPP plugins statuses

Closes: #7603


### Description


Adds the following tags depending on the status of the plugins to **all types of tickets**:

##### WCPay
* `woo_mobile_wcpay_not_installed`
* `woo_mobile_wcpay_installed_and_not_activated`
* `woo_android_wcpay_installed_and_activated`
##### Stripe
* `woo_mobile_stripe_not_installed`
* `woo_mobile_stripe_installed_and_not_activated`
* `woo_mobile_stripe_installed_and_activated`

If fetching the statuses of the plugins fails, we add:
* `woo_mobile_site_plugins_fetching_error`

### Testing instructions

* Go to the settings
* Help & Support
* Click on `Contact support`
* Create a test ticket and notice that the tags described above are attached to the ticket 
---
* Go to the settings
* Help & Support
* Click on `Contact Payments Support`
* Create a test ticket and notice that the tags described above are attached to the ticket 
---
* Go to the settings
* Help & Support
* Disable internet
* Click on `Contact Payments Support`
* Create a test ticket and notice `woo_android_site_plugins_fetching_error` tag is attached
---
* Modify `HelpActivity::110` to 

```
        if (true) {
            viewModel.contactSupport(TicketType.General)
        }
```
* Go to the settings
* Help & Support
* Create a test ticket and notice that the tags described above are attached to the ticket 


### Images/gif


<img width="374" alt="image" src="https://user-images.githubusercontent.com/4923871/197769940-d6af3153-5f74-485f-96f2-4f70ca2cdaee.png">

https://user-images.githubusercontent.com/4923871/197761621-685b4407-c5a5-4ba7-98ee-f6058b7f6968.mp4
